mmv1/templates/terraform/examples/cloudquotas_quota_preference_basic.tf.erb

```diff
@@ -0,0 +1,15 @@
+# [START cloudquotas_quota_preference_basic]
```

Comment:
These tags are no longer needed, so they can be removed

Reply:
Done.